androidApp/src/main/java/dev/johnoreilly/confetti/ui/ConfettiScaffold.kt

```diff
@@ -74,10 +77,20 @@ fun ConfettiScaffold(
                         containerColor = Color.Transparent
                     ),
                     actions = {
+                        val viewModel: AccountViewModel = getViewModel()
```

Comment:
What about introducing a stateful and stateless pair? Similar to https://developer.android.com/jetpack/compose/state#state-hoisting

It keeps this class simpler.

You just end up having the original stateful form of AccountIcon, but then delegate all UI structure to the stateless form.  Then test and preview that.

Reply:
Aha, yeah, that sounds nice! I can try that out! That means that I can revert the changes in the scaffold component to right?😊

Reply:
Yep.